src/test/java/com/rackspace/salus/monitor_management/services/MonitorManagementExcludeResourceIdsTest.java

```diff
@@ -81,6 +84,7 @@
 @AutoConfigureDataJpa
 @AutoConfigureTestEntityManager
 @Transactional
+@Import({MetricsAutoConfiguration.class, CompositeMeterRegistryAutoConfiguration.class})
```

Comment:
I usually just import `SimpleMeterRegistry` to get both a real instance and not all of the auto-configured exports that we might happen to have in our apps (however, they're disabled by default).
```suggestion
@Import({SimpleMeterRegistry.class})
```